### PR TITLE
change commitment message recent to processed 

### PR DESCRIPTION
**Pull request by Necmttn**
Cleanup for deprecated commitment messages. referred in here: https://github.com/project-serum/anchor/issues/289
checked other deprecated messages did not find any:
https://github.com/solana-labs/solana-web3.js/blob/45923ca00e4cc1ed079d8e55ecbee83e5b4dc174/src/connection.ts#L230

### CODE REVIEW

**Comment by paul-schaaf** [comment]
@Necmttn thanks for the PR! There is at least one `max` in the codebase (in the swap tests utils js file) that you can also remove. 

pls also add a line under `Unreleased -> Fixes` in the changelog

**Comment by Necmttn** [comment]
@paul-schaaf updated. ✌️

File: tests/swap/tests/utils/index.js [hunk 1]
```diff
@@ -457,7 +457,7 @@ async function signTransactions({
   wallet,
   connection,
 }) {
-  const blockhash = (await connection.getRecentBlockhash("max")).blockhash;
+  const blockhash = (await connection.getRecentBlockhash("processed")).blockhash;
```
Comment (paul-schaaf) [inline tests/swap/tests/utils/index.js:460]: finalized, not processed

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ incremented for features.
 * cli: Add version number from programs `Cargo.toml` into extracted IDL ([#1061](https://github.com/project-serum/anchor/pull/1061)).
 * lang: Add `deprecated` attribute to `Loader`([#1078](https://github.com/project-serum/anchor/pull/1078)).
 * lang: the `init_if_needed` attribute now checks that given attributes (e.g. space, owner, token::authority etc.) are validated even when init is not needed ([#1096](https://github.com/project-serum/anchor/pull/1096)).
+* lang: Change commitment message recent to processed ([#1128](https://github.com/project-serum/anchor/pull/1128))
```
Comment (paul-schaaf) [inline CHANGELOG.md:22]: this msg is in the wrong place. `0.19.0` is released already. you can add this to a new `fixes` section in the `Unreleased` section

**Comment by Necmttn** [comment]
Checked the failing test but seems like it's not related to this pr?

**Comment by paul-schaaf** [comment]
merge master again @Necmttn

File: ts/src/utils/rpc.ts [hunk 3]
```diff
@@ -72,6 +72,7 @@ async function getMultipleAccountsCore(
 ): Promise<
   Array<null | { publicKey: PublicKey; account: AccountInfo<Buffer> }>
 > {
+
```
Comment (paul-schaaf) [inline ts/src/utils/rpc.ts:75]: ^ line can be deleted

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,8 +11,9 @@ incremented for features.
 
 ## [Unreleased]
 
-### Fixes
 
+### Fixes
+* lang: Change commitment message recent to processed ([#1128](https://github.com/project-serum/anchor/pull/1128))
```
Comment (paul-schaaf) [inline CHANGELOG.md:16]: also max -> finalized

**Review by paul-schaaf — APPROVED** [review]

**Pull request merged**